README.md

```diff
@@ -52,7 +52,7 @@ Supports
 Configuration
 ---
 
-A very small amount of configuration is possible by creating a file called `leech.json` in the project directory. Currently you can define login information for sites that support it, and some options for book covers.
+A very small amount of configuration is possible by creating a file called `leech.json` in your current directory. Currently you can define login information for sites that support it, and some options for book covers.
```

Comment:
I haven't actually tested the `leech.json` configuration. 😟 I thiiiink, the way it's written now, it'll look in your current working directory?